exercises/practice/resistor-color/resistor_color.asm

```diff
@@ -1,3 +1,7 @@
+default rel
```

Comment:
If we omit this line from the example solution, we see
```
/usr/bin/ld: resistor_color.o: relocation R_X86_64_32S against `.data' can not be used when making a PIE object; recompile with -fPIE                                     
/usr/bin/ld: failed to set dynamic section sizes: bad value                          
collect2: error: ld returned 1 exit status                                           
make: *** [Makefile:32: tests] Error 1
```

That error does not guide people to add `default rel`